src/components/cluster/new/CreateRegularCluster.js

```diff
@@ -331,9 +340,51 @@ class CreateRegularCluster extends React.Component {
     });
   };
 
+  produceRAMAndCoresAWS = () => {
+    const { awsInstanceTypes } = this.state;
+    const instanceType = this.state.aws.instanceType.value;
```

Comment:
Because of this: https://github.com/giantswarm/fmt/blob/master/javascript/general_style_guide.md#props-and-state--es6-destructuring

I will destructure anything that is used more than once, as a general rule.

Reply:
Feel free to open a PR for this if you want. As I told you, this style guide is a WIP,  not set in stone, at all.